dScripts/02_server/Map/AM/AmBlueX.cpp

```diff
@@ -18,7 +18,7 @@ void AmBlueX::OnSkillEventFired(Entity* self, Entity* caster, const std::string&
 
 		auto* character = caster->GetCharacter();
 		if (character != nullptr) {
-			character->SetPlayerFlag(self->GetVar<int32_t>(m_FlagVariable), true);
+			character->SetPlayerFlag(self->GetVar<uint32_t>(m_FlagVariable), true);
```

Comment:
Can you confirm this still works with the above knowledge that flags are ints and not unsigned ints?